### PR TITLE
Feature: Add project expansion item

### LINKED ISSUE #251: HomePage: Expansion panel for project in drawer
## Description

Implements the expansion panel "Projects":
- Only 4 projects are listed
- Sort project by A-Z
- Take the first four project
- each project will redirect to the index page of project

Not implements:
- the "see all" button

Expected design:
image
image

### PR DESCRIPTION
**Pull request by justinehell**
Resolve #251

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -10,7 +10,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 * Homepage:
-  * Display left drawer.
+  * Display left drawer
```
Comment (Zorin95670) [inline changelog.md:13]:
```
* Display a left drawer that contains:
  * A button to display recents projects
  * A short project list access
```

Comment (justinehell) [reply changelog.md]: done

**Review by Zorin95670 — DISMISSED** [review]

File: changelog.md [hunk 2]
```diff
@@ -10,7 +10,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 * Homepage:
-  * Display left drawer.
+  * Add a left drawer that contains:
+    * A button to display recent projects.
+    * A short project list access.
```
Comment (k-y0u) [inline changelog.md:15]: `project list` here, `projects list` in e2e tests... choose one

**Review by k-y0u — CHANGES_REQUESTED** [review]

File: src/components/list/ShortProjectList.vue [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+  onUnmounted,
+} from 'vue';
+import { getProjects } from 'src/composables/Project';
+import ProjectEvent from 'src/composables/events/ProjectEvent';
+
+const projects = ref([]);
+let updateProjectSubscription;
+
+/**
+ * Initialize project list with the first four projects in alphabetical order.
```
[the remark this answers is not in the document]
Comment (justinehell) [reply src/components/list/ShortProjectList.vue:51]: done

File: src/components/list/ShortProjectList.vue [hunk 4]
```diff
@@ -0,0 +1,68 @@
+<template>
+  <q-list class="short-project-list">
```
[the remark this answers is not in the document]
Comment (justinehell) [reply src/components/list/ShortProjectList.vue]: updated

**Review by Zorin95670 — DISMISSED** [review]

**Review by Zorin95670 — APPROVED** [review]

**Pull request merged**